client/tsconfig-reference.json

```diff
@@ -0,0 +1,6 @@
+{
+	"extends": "./tsconfig.json",
+	"compilerOptions": {
+		"noEmit": false
```

Comment:
A project has to set `noEmit` to `false` to be able to be referenced, because the project would be build independently. 

https://www.typescriptlang.org/tsconfig#noEmit